package.json

```diff
@@ -40,7 +40,10 @@
     "url": "https://github.com/code-and-conquer/darwin/issues"
   },
   "homepage": "https://github.com/code-and-conquer/darwin#readme",
-  "dependencies": {},
+  "dependencies": {
+    "@monaco-editor/react": "^3.1.2",
+    "monaco-editor": "^0.20.0"
```

Comment:
That's the wrong package.json. This must be in `game-frontend/package.json`. (Small hint: look for the package.json containing other runtime dependencies such as react)

Reply:
is that the reason eslint was complaining?

Reply:
Yes, usually when you get an error there is a reson - that's the reason 😄 
Add the dependencies you want to `game-frontend/package.json` and you're good to go.

Btw: As far as I'm aware of you don't have to add `monaco-editor` at all :v: